aiven/client/cli.py

```diff
@@ -3737,7 +3758,7 @@ def vpc__peering_connection__delete(self):
             print(ex.response.text)
             raise
 
-    def _get_service_project_vpc_id(self):
+    def _get_service_project_vpc_id(self) -> Optional[Union[object, str]]:
```

Comment:
Probably can be `Union[str, None, Literal[UNDEFINED]]` instead.

Reply:
`Literal` is only in py3.8+